data_structures/linked_list/singly_linked_list.py

```diff
@@ -4,52 +4,60 @@ def __init__(self, data):
         self.next = None
 
     def __repr__(self):
-        return f"Node({self.data})"
+        return f"{self.data}"
```

Comment:
Please undo.  The [`__repr__()` code was correct](https://docs.python.org/3/reference/datamodel.html#object.__repr__) before this change.
> this should look like a valid Python expression that could be used to recreate an object with the same value